docs/docsite/rst/dev_guide/developing_collections_distributing.rst

```diff
@@ -175,12 +184,90 @@ For example, to exclude the :file:`sensitive` folder within the ``playbooks`` fo
      - playbooks/sensitive
      - '*.tar.gz'
 
-For more information on the :file:`galaxy.yml` file, see :ref:`collections_galaxy_meta`.
-
 .. note::
      The ``build_ignore`` feature is only supported with ``ansible-galaxy collection build`` in Ansible 2.10 or newer.
 
 
+.. _manifest_directives:
+
+Manifest Directives
+^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.14
+
+The :file:`galaxy.yml` file supports manifest directives that are historically used in Python packaging, as described `here <https://packaging.python.org/en/latest/guides/using-manifest-in/#manifest-in-commands>`_.
```

Comment:
```suggestion
The :file:`galaxy.yml` file supports manifest directives that are historically used in Python packaging, as described in `MANIFEST.in commands <https://packaging.python.org/en/latest/guides/using-manifest-in/#manifest-in-commands>`_.
```

Reply:
^^ bit of a nit but accessibility guidelines say we should avoid 'here' as a link and try to describe where the link is going instead. Helps for those using screen readers.